aci/resource_aci_l3extlifp.go

```diff
@@ -60,7 +60,7 @@ func resourceAciLogicalInterfaceProfile() *schema.Resource {
 				Optional: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"tn_netflow_monitor_pol_name": {
+						"tn_netflow_monitor_pol_dn": {
 							Type:     schema.TypeString,
 							Required: true,
 						},
```

Comment:
flt_type can be validated instead of string, options as displayed in model-doc are:

CE type | 3 | ce | CE filter type
- | ipv4 | defaultValue | NO COMMENTS
Ipv4 type | 1 | ipv4 | Ipv4 filter type
Ipv6 type | 2 | ipv6 | Ipv6 filter type

Reply:
Done!